NavigationReactNative/src/android/app/src/main/java/com/navigation/reactnative/TabLayoutManager.java

```diff
@@ -35,6 +35,12 @@ public void setSelectedIndicatorAtTop(TabLayoutView view, boolean selectedIndica
         view.setSelectedTabIndicatorGravity(!selectedIndicatorAtTop ? TabLayout.INDICATOR_GRAVITY_BOTTOM : TabLayout.INDICATOR_GRAVITY_TOP);
     }
 
+    @ReactProp(name = "scrollable")
+    public void setTabScrollable(TabLayoutView view, @Nullable boolean scrollable) {
+        view.setTabMode(scrollable ? TabLayout.MODE_SCROLLABLE : TabLayout.MODE_FIXED);
```

Comment:
We don't need to `setTabMode` here because the view takes care of that now. Also can we change the name to `setScrollable` instead of `setTabScrollable`?

Reply:
Ok